render/tests/test-attributes.js

```diff
@@ -131,6 +131,66 @@ o.spec("attributes", function() {
 			o(a.dom.attributes["class"].nodeValue).equals("test")
 		})
 	})
+	o.spec("class and className", function() {
+		o("setting, updating and removing with class", function() {
+			var a = {tag: "div", attrs: {class: "test"}}
+			var b = {tag: "div", attrs: {class: "toast"}}
+			var c = {tag: "div", attrs: {}}
+
+			render(root, [a]);
+
+			o(a.dom.attributes["class"].nodeValue).equals("test")
```

Comment:
You don't need to quote these - IE9 won't choke over them, and some of the member accesses already use `.class`.